structuralcodes/sections/_rc_utils.py

```diff
@@ -0,0 +1,84 @@
+from structuralcodes.geometry import CompoundGeometry, SurfaceGeometry
+from structuralcodes.materials.constitutive_laws import Elastic, UserDefined
+from structuralcodes.sections import GenericSection
+
+
+def calculate_elastic_cracked_properties(section: GenericSection, theta=0):
+    """Calculates the cracked section properties of a reinforced concrete
+    section.  (GenericSection). Materials in surface geometries and point
+    geometries are elastic-linear  in order to make the cracking properties
+    independent of the stress state.  Tension in all surface geometries is
+    neglected.
+
+    Args:
+        section: GenericSection
+        theta: Angle of the neutral axis to the horizontal. theta=0 implies
+               upper compression block.
+
+    Returns:
+        cracked_prop : GrossProperties data of cracked_sec (i.e cracked
+                       section properties)
+        cracked_sec: GenericSection removing tension areas in geometries.
+    """
+
+    def create_surface_geometries(polygons_list, material):
+        """Process shapely polygons to SurfaceGeometries."""
+        # Create an empty list to store SurfaceGeometry objects
+        surface_geometries = []
+
+        # Iterate over the list of polygons and create SurfaceGeometry for each
+        for polygon in polygons_list:
+            # Create a new SurfaceGeometry for the current polygon
+            surface_geometry = SurfaceGeometry(polygon, material)
+            # Add the new SurfaceGeometry to the list
+            surface_geometries.append(surface_geometry)
+
+        return surface_geometries
+
+    if not section.geometry.reinforced_concrete:
+        return None
+
+    rotated_geometry = section.geometry.rotate(theta)
```

Comment:
I think we should rotate by -theta? see e.g. calculate bending strength method in the generic section.

The reason is because if we want to assess the inclination of n.a. equal to `theta` , to have n.a. horizontal, we should rotate the section by `-theta`:

![image](https://github.com/user-attachments/assets/3a35f328-13fd-4a52-9627-0a0d9ee0fea5)

Reply:
Better. This way, we maintain sign consistency within the package.